context/app/static/js/shared-styles/DialogModal/DialogModal.jsx

```diff
@@ -30,10 +31,14 @@ function DialogModal({
         </Typography>
       </StyledDialogTitle>
       <DialogContent>
-        {errorMessage && (
-          <Alert $marginBottom={20} severity="error">
-            {errorMessage}
-          </Alert>
+        {errorMessages && (
+          <Box sx={{ marginBottom: 3 }}>
+            {errorMessages.map((errorMessage) => (
+              <Alert $marginBottom={10} severity="error">
+                {errorMessage}
+              </Alert>
+            ))}
+          </Box>
```

Comment:
I created a separate Box container around the error messages for proper spacing. 

![image](https://github.com/hubmapconsortium/portal-ui/assets/45410262/16716d96-6a20-4da0-abd1-1e80ca52e4ea)

Reply:
This looks pretty good! A few minor things:

 - Components rendered in a `map` need a `key` to be provided for React to be able to tell apart the different instances. I suggest using the error message itself as the key.
 - This is completely optional, but I wonder if adding `display: 'grid' and `gap: 1.25` (i.e. 1.25 * 8 = 10px) to the Box and removing `$marginBottom={10}` would also work to manage the spacing? https://mui.com/system/getting-started/the-sx-prop/#grid